_posts/2021-02-01-coalesce-dbt-summary.markdown

```diff
@@ -0,0 +1,288 @@
+
+Carly gives a great backgrounder on the industry and experiences of working in ETL that many people will relate to, with tools like Informatica. Equally, she illuminates many of the basic challenges that DBT helps to address, including adopting many of the practices in data work that are typically commonplace now in Software Engineering. This video is an excellent starting place for anyone looking to understand what DBT does.
+
+### The Post Modern Data Stack
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/C_R6VHJfQn4" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+In this broadly focused video, Drew draws parallels between period of styles art, and the evolution of data processing. It's a great experience in zooming out and seeing the bigger picture - in this case, how the decomposition of data platforms in to stacks of more de-coupled components, focused more around the teams and capabilities organisations have, the challenges for which customers are truly seeking answers, and the capability to adapt to change and advancements in technology.
+
+### How JetBlue became a Data-Driven airline using DBT
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/MlQGHR5bvRI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+This video is a clear favourite for me, simply for the sheer volume and depth of insight it provides. Ashley covers a multitude of topics that almost every organisation on a data journey will encounter at one time or another:
+* Defining what it means to be *data-driven*
+* Identifying what your modern data stack should look like
+* Why a Transition Plan is a fundamental keystone to the success of any large data migration
+* How DBT helped JetBlue significantly increase their internal contributor rate towards data work
+
+Amongst these topics were many other themes including the importance of engaging for change early within an organisation, investing in the training of your people, and the commitment to sunsetting your legacy systems. Check it out!
+
+
+### How JetBlue secures and protects data with DBT and Snowflake
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/NPQOTm1XITw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Everyone at some point will need to deal with privacy requirements in a data warehouse, and this shorter talk gives a great overview of JetBlue's iterative journey to tackling this challenge. To me, the greatest part of the story is how the goal is achieved latterly by policy, as opposed to processing. 
+
+
+### Organisational Epistemelogy - how do we know stuff?
```

Comment:
"how do we know stuff?" should be "how do we know what is true?"

Reply:
it's actually the full title of the presentation